.github/workflows/clp-dependency-image-build.yaml

```diff
@@ -26,9 +26,23 @@ jobs:
 
       - name: Workaround actions/runner-images/issues/6775
         run: chown $(id -u):$(id -g) -R .
+
+      - name: Filter relevant changes
+        uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          # Consider changes between the previous commit and this one
+          base: ${{ github.ref_name }}
+          filters: |
+            src:
+              - '.github/workflows/clp-dependency-image-build.yaml'
+              - 'components/core/tools/docker-images/clp-env-base-focal/**'
+              - 'components/core/tools/scripts/lib_install/*.sh'
+              - 'components/core/tools/scripts/lib_install/ubuntu-focal/**'
```

Comment:
Bring up a discussion here:
it seems repeated to have these paths specified in three builds. Is it possible to make the path (at least the prefix) as variables (like in `env`)?
Seems like there are some advanced uses such as defining a shell function to generate the paths and make the target OS as the input variable. Not sure if we should go to that level, lol

Reply:
Replaced the prefixes